apisix/admin/plugins.lua

```diff
@@ -125,10 +124,6 @@ end
 
 function _M.get_plugins_list()
     local plugins = core.config.local_conf().plugins
-    if plugins[1] == 'example-plugin' then
```

Comment:
@juzhiyuan I’m not sure if delete it will affect the display of the front-end, for example, this plugin will be displayed on the interface

Reply:
No, it will not effect the frontend any more.